ftw/upgrade/jsonapi/utils.py

```diff
@@ -24,6 +25,14 @@
 import transaction
 
 
+try:
```

Comment:
Maybe use `pgk_resources` to detect if `plone.protect` is installed. Otherwise if the interface should be moved we don't disable csrf even if `plone.protect` is installed.

try:
    pkg_resources.get_distribution('plone.protect')
    from plone.protect.interfaces import IDisableCSRFProtection
except pkg_resources.DistributionNotFound:
    ...
```